equipmanager.lic

```diff
@@ -151,6 +151,18 @@ class EquipmentManager
     end
   end
 
+  def wield_weapon_offhand(description)
```

Comment:
I noticed that `wield_weapon` has a special case for the Offhand skill. Is that a responsibility of the equipment manager, or should combat-trainer be figuring that out and using one of these two `wield*` functions? This isn't part of the PR, but maybe food for thought...